docs/tutorials/analysis/1D/spectral_analysis_rad_max.ipynb

```diff
@@ -0,0 +1,527 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    safe_mask_masker = SafeMaskMaker(methods=["aeff-max"], aeff_percent=10)
</pre><p>Is this choice adapted to MAGIC data? How is the low threshold determined usually?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy/pull/3712/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
No, we calculate the energy threshold either from the MC distribution after analysis cuts, or with a simple formula obtained in the performance paper. This is stored in the `LO_THRES` keyword in the `AEFF` header.
How do I use it, isn't it used automatically?

Reply:
Normally the `aeff-default` method of the `SafeMaskMaker` should be fine.

Reply:
I am using it now.